proto/v1/external_vcs_service.proto

```diff
@@ -0,0 +1,202 @@
+syntax = "proto3";
+
+package bytebase.v1;
+
+import "google/api/annotations.proto";
+import "google/api/client.proto";
+import "google/api/field_behavior.proto";
+import "google/protobuf/empty.proto";
+import "google/protobuf/field_mask.proto";
+
+import "v1/common.proto";
+
+option go_package = "generated-go/v1";
+
+service ExternalVCSService {
+  rpc CreateExternalVCS(CreateExternalVCSRequest) returns (ExternalVCS) {
+    option (google.api.http) = {
+      post: "/v1/externalVCS"
```

Comment:
Do we have the naming pattern for the path? For example, `externalVCS` or `external-vcs` or `external_vcs`
And how about using `/v1/gitops/{type}/repositories/{repository}`? Then we can use `/v1/gitops/-/repositories` to list all repositories
@d-bytebase @h3n4l

Reply:
IMO, our v1 API is a resource-oriented design. I am unsure whether we can define `gitops` as a resource.